shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -159,6 +159,8 @@ void Reset() {
  */
 @interface FlutterViewWrapper : NSView
 
+- (void)setBackgroundColor:(NSColor*)color;
```

Comment:
This isn't pretty, but the `FlutterViewController`'s content view is a `FlutterViewWrapper` and not a `FlutterView`. AFAIK, this is the only way to route the user's intent to the `FlutterView`/

> cc @chunhtai

Reply:
The FlutterViewController should have direct access to FlutterView?

Reply:
The `FlutterViewController`'s view is set to a wrapped `FlutterView`
https://github.com/flutter/engine/blob/e39c4dfe72bc64e3a293cdb066e9cc788478c14f/shell/platform/darwin/macos/framework/Source/FlutterViewController.mm#L379-L380

but the `FlutterViewWrapper`'s `FlutterView` is private. 
https://github.com/flutter/engine/blob/e39c4dfe72bc64e3a293cdb066e9cc788478c14f/shell/platform/darwin/macos/framework/Source/FlutterViewController.mm#L255-L257

I tried calling `setBackgroundColor` on `self.view` (`FlutterView`) from `FlutterViewController` by casting `self.view` to a `FlutterView`. Oddly enough there was no runtime error, but the call took no effect until I routed the call through `FlutterViewWrapper`.

Reply:
I meant the FlutterViewController should be able to access flutter view directly https://github.com/flutter/engine/blob/7d5df8106a9a278714674a75b9736f60066e32ba/shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h#L14

Reply:
It's probably PEBKAC